test/powershell/Modules/Microsoft.PowerShell.Utility/Set-Date.Tests.ps1

```diff
@@ -1,14 +1,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe "Set-Date for admin" -Tag @('CI', 'RequireAdminOnWindows') {
+Describe "Set-Date for admin" -Tag @('CI', 'RequireAdminOnWindows', 'RequireSudoOnUnix') {
     # Currently, CI tests on Linux/macOS are always run as normal user. So we need to skip these tests on non-Windows platform.
```

Comment:
the comments here should be updated

Reply:
I added a specific comment to address this https://github.com/PowerShell/PowerShell/pull/6145#pullrequestreview-96294299